stack/eslint-plugin-security/package.json

```diff
@@ -1,39 +1,39 @@
 {
```

Comment:
Intentional indent changes?

Reply:
Something is reformatting and sorting these package.json files. I think it might be Prettier. (?)

Reply:
It's PNPM. (Thanks Process Monitor!)  `pnpm pack` rewrites the package.json file before invoking `npm pack` and does not restore it afterwards.  😬